src/ybox/conf/distros/neon/init.sh

```diff
@@ -0,0 +1 @@
+#!/bin/sh -e
```

Comment:
Edit about mirrors:

I tried running netselect and it does not work without real root privilege which will not work in rootless docker/podman. So far the best option seems to be apt-select (e.g. https://www.letscloud.io/community/how-to-find-best-ubuntu-apt-repository-mirror). The following seems to work fine for me, though it may not work in some countries that have no mirrors so for that there is a fallback to US:

```sh
apt-get install curl libsasl2-modules libldap-common publicsuffix python3-requests python3-soupsieve python3-charset-normalizer python3-pip -y
pip3 install apt-select
if ! apt-select --country $(curl -sSL http://ipinfo.io/country); then
  apt-select --country US
fi
if [ -f sources.list ]; then
  mv /etc/apt/sources.list /etc/apt/sources.list.bak && cp sources.list /etc/apt/sources.list && rm -f sources.list
fi
apt-get update
apt-get clean
```

For ubuntu 24.04, move /etc/apt/sources.list.d/ubuntu.sources to a bak file.

Reply:
Unfortunately the problem with many mirrors is their unreliability. In my testing just now, apt-select selected mirrors.piconets.webwerks.in which has a very good speed. But installing packages shows unresolvable broken dependencies meaning the mirror is only partially updated. It will get fixed in a few hours, but this causes me to always revert back to basic servers like in.archive.ubuntu.com and use [apt-fast](https://github.com/ilikenwf/apt-fast) for installs.

So I guess the other option is to select a decent generic mirror like us.archive.ubuntu.com and configure apt-fast+aria2 which gives overall the best experience for me. Something like:

```sh
export DEBIAN_FRONTEND=noninteractive
apt-get install curl lsb-release gnupg -y
rel_name=$(lsb_release -cs)

cat > /etc/apt/sources.list << EOF
deb http://us.archive.ubuntu.com/ubuntu/ $rel_name main restricted universe multiverse
deb http://us.archive.ubuntu.com/ubuntu/ $rel_name-updates main restricted universe multiverse
deb http://us.archive.ubuntu.com/ubuntu/ $rel_name-backports main restricted universe multiverse
deb http://us.archive.ubuntu.com/ubuntu/ $rel_name-security main restricted universe multiverse
EOF

echo "deb [signed-by=/etc/apt/keyrings/apt-fast.gpg] http://ppa.launchpad.net/apt-fast/stable/ubuntu $rel_name main" > /etc/apt/sources.list.d/apt-fast.list
mkdir -p /etc/apt/keyrings
curl -fsSL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xA2166B8DE8BDC3367D1901C11EE2FF37CA8DA16B" | gpg --dearmor -o /etc/apt/keyrings/apt-fast.gpg
apt-get update
apt-get install aria2 apt-fast -y
```

Then use apt-fast for installations in distro.ini pkgmgr section, and even in the init scripts like:

DOWNLOADBEFORE=true apt-fast install locales less man-db manpages bash-completion git patch ed bc unzip fastjar wget curl gnupg openssh-client shared-mime-info ubuntu-minimal iso-codes mesa-utils vainfo intel-media-va-driver-non-free va-driver-all vulkan-tools mesa-vulkan-drivers ca-certificates python3-pip pulseaudio-utils libsasl2-modules libldap-common manpages xdg-user-dirs xauth libgl1-amber-dri -y